include/logging/log.h

```diff
@@ -424,8 +424,7 @@ static inline char *log_strdup(const char *str)
  *
  */
 #define LOG_LEVEL_SET(level) \
-	static const u32_t __log_level __attribute__((unused)) = \
-			_LOG_LEVEL_RESOLVE(level)
+	static const u32_t __log_level __attribute__((unused)) = level
```

Comment:
Would be nice to use `__unused` instead of `__attribute__` directly for toolchain independence while you are here.

Reply:
Looks good, thanks. Needs rebase.

Reply:
done